File: fundamentos/calculadora.py
def suma():
    valor1 = 10
    valor2 = 20
    suma = valor1 + valor2
    print(f'La suma es igual a: {suma}')
    
def resta():
    valor1 = 10
    valor2 = 20
    resta = valor1 - valor2
    print(f'La resta es igual a: {resta}')
    
def multiplicacion():
    x=float(input('Ingrese un multiplicador'))
    y=float(input('Ingrese un valor'))
    product=x*y
    print('El producto de ', x, '*', y, 'es ', product)
    
def division():
    valor1 = 24
    valor2 = 8
    divis = valor1 / valor2
    print(f'La división es igual a: {divis}')
    
def calculadora():
    suma()
    resta()
    multiplicacion()
    division()
    
calculadora()

File: practicas/js02/practicas/js02arantxa/calculadora.js
function sumar() {
    const valor1 = parseInt(document.getElementById('numero1').value);
    const valor2 = parseInt(document.getElementById('numero2').value);

    const suma = valor1 + valor2;

    const resultado = document.getElementById('resultado');
    resultado.innerHTML = `Resultado: ${suma}`;
}

function resta() {
    const valor1 = parseInt(document.getElementById('minuendo').value);
    const valor2 = parseInt(document.getElementById('sustraendo').value);

    const resta = valor1 - valor2;

    const diferencia = document.getElementById('diferencia');
    diferencia.innerHTML = `Resultado: ${resta}`;
}

function multiplicacion() {
    const valor1 = parseInt(document.getElementById('multiplicando').value);
    const valor2 = parseInt(document.getElementById('multiplicador').value);

    const multiplicacion = valor1 * valor2;

    const producto = document.getElementById('producto');
    producto.innerHTML = `Resultado: ${multiplicacion}`;
}

function division() {
    const valor1 = parseInt(document.getElementById('dividendo').value);
    const valor2 = parseInt(document.getElementById('divisor').value);
    
    const resultadoDiv = document.getElementById('cociente');

    if (valor2 === 0) {
        resultadoDiv.innerHTML = 'La división entre cero no es posible, ingrese otro número';
    } else {
        const resultado = valor1 / valor2; 
        resultadoDiv.innerHTML = `Resultado: ${resultado}`;
    }
}